api/views_export.py

```
@@ -0,0 +1,24 @@
+    if not request.user.is_authenticated:
+        return JsonResponse({"error":"unauthorized"}, status=401)
+    try:
+        p = Patient.objects.get(pk=pk)
+        data = {
+            "patient": {"id": str(p.id), "name": p.full_name},
+            "encounters": list(Encounter.objects.filter(patient=p).values()),
+            "labs": list(LabResult.objects.filter(patient=p).values()),
+            "medications": list(MedicationOrder.objects.filter(patient=p).values()),
+            "summaries": list(AISummary.objects.filter(patient=p).values()),
+        }
+        return JsonResponse(data, safe=False, json_dumps_params={'cls':DjangoJSONEncoder})
```

Ruff (0.12.2)

Line too long (90 > 88)

(E501)